src/fundamentals-1.md

```diff
@@ -143,6 +148,9 @@ Add HTML5 compliant headlines and sections to structure the content of your page
 #### Inner values matter!
 Add dummy text and image content to your page, you can use dummy text and image generators for that.
 With latter you can easily create dummy images like this:
+
+---- Text should be above the img ----
```

Comment:
Are these `----` prefixed sentences "comments"? I'm not sure how to interpret them

Reply:
yes they are comments, because i didnt know how to change it by myself :)